README.md

```diff
@@ -49,6 +49,85 @@ public
 Ressource: 
 - [reate-new-express-js-apps-with-express-generator](https://www.sitepoint.com/create-new-express-js-apps-with-express-generator/)
 
+# Deploying to Google Cloud
+
+1. Installing Google Cloud SDK 
+
+- Source for [installation](https://cloud.google.com/sdk/docs/quickstart#deb)
+  - Note: Be sure to choose the right Operating System
+```
```

Comment:
Instead of putting 3 bash commands in 3 code blocks I would put all 3 commands into one code block, if needed you can add a comment. Like this:
```
# comment 1
command 1
# comment 2
command 2
```

Reply:
Changed to one code block.